content/organization/intro/co-organizers/index.en.md

```diff
@@ -33,21 +33,31 @@ You can point people who hesitate to jump on board to the [intro chapter for org
 If some people are happy to help without committing, it's already a great progress for you.
 Remind them regularly they could become an official organizer. :wink:
 
-## How to onboard organizers?
+## How to onboard new organizers?
 
-* before a volunteer commits as an organizer, an idea might be to have them host (i.e. organize) one or a few meetups (be careful to acknowledge their work although they're not an official organizer (yet)!) --- see [Meetup docs about the "event organizer" role](https://help.meetup.com/hc/en-us/articles/360002879411-Managing-a-leadership-team);
+* Before a volunteer commits as an organizer, an idea might be to have them host (i.e. organize) one or a few meetups (be careful to acknowledge their work although they're not an official organizer (yet)!) --- see [Meetup docs about the "event organizer" role](https://help.meetup.com/hc/en-us/articles/360002879411-Managing-a-leadership-team);
 
-* after that, register them as organizers 
-  * on meetup and 
-  * [at the global level](https://github.com/rladies/rladiesguide/issues/18) (guidance in flux) to have them access your chapter email address; get invited to the organizers slack. 
-  * Locally, make sure to give them access to the needed tools/services e.g. Tweetdeck access to Twitter.
+* After that:
+  * Send new organizers the [R-Ladies Organizers form](https://rladies.org/form/organiser).
```

Comment:
Hello everyone,
We have the following private issue ( https://github.com/rladies/new-chapters-onboarding/blob/main/.github/ISSUE_TEMPLATE/existing-chapter-update.md) for tracking updates on existing chapters. @cosimameyer, I'm not sure if you have access to it. We could suggest that the current organizer either follow the steps mentioned to onboard new organizers or contact the chapters@rladies.org team. In that case, the chapter onboarding team can handle the process by opening the issue and tracking the necessary steps.
I don’t think it makes sense to notify the chapter onboarding team just to open the issue if the current organizers feel comfortable following the guide and completing the steps themselves. Or maybe it's up to the chapter onboarding team to open an issue and handle these steps regardless? In that case, this section should be presented differently, without requiring the current organizer to go through the onboarding steps for new organizers. What do you think?

@acolum please feel free to add your thoughts if needed!

Reply:
The process that @acolum described in the issue template seems very well organized. We can ask the new organizer to update the information on GitHub themselves if they feel comfortable doing so or request help from someone on the team. That can be discussed when they contact us.
I'm not sure if contacting the onboarding team should be presented as optional. We need people to inform us about upcoming changes in their chapters so we can open an issue and track all the necessary steps. Currently, many chapters have added organizers on Meetup but haven't filled out the form, added the organizer to GitHub, etc. If they don’t use Slack, we have no way to contact them.
Maybe we could take a minimal approach here: just telling people that for onboarding and retiring organizers they need to contact us via email. That wil also make easier to maintain these parts of the Guide.  
In the [Coordination](https://guide.rladies.org/coordination/onboarding/) section we can describe the process of opening an issue and any other step we need to consider. In this part, which is intended for organizers, it might be better to keep things simple and straightforward

Reply:
Thank you so much for your feedback, @rivaquiroga. I understand what you mean and agree with your suggestion. I'll move the details I've added about onboarding and retiring organizers to the Coordination section, and keeping the process simple in the Organizers section by directing the organizers to contact the onboarding team to onboard or retire organizers.